authors/apps/profiles/views.py

```diff
@@ -83,13 +85,20 @@ def patch(self, request, username):
         )
 
 
-class FollowAPI(GenericAPIView):
+class UserNotFound(APIException):
+    """exception message"""
+    status_code = 404
+    default_detail = 'User with that username Not found'
+
```

Comment:
save exceptions in an exceptions file to make it reusable

Reply:
@graycadeau thank you for the feedback let me work on that

Reply:
@graycadeau I have made the exception file reusable